Dockerfile

```diff
@@ -28,6 +31,10 @@ RUN curl -sL -o restic.tar.gz https://github.com/restic/restic/releases/download
  && cd .. \
  && rm restic.tar.gz restic-${RESTIC_VERSION} -fR
 
+RUN curl -O https://downloads.rclone.org/rclone-current-linux-amd64.zip \
+ && unzip -a rclone-current-linux-amd64.zip \
+ && mv rclone-*/rclone /usr/local/bin/rclone
```

Comment:
Is it possible to download a fixed version and check the checksum, as we do with go-cron? Of course the `RCLONE_VERSION` should be a build arg as well.